src/test/kotlin/model/lotto/LottoNumberTest.kt

```diff
@@ -0,0 +1,24 @@
+package model.lotto
+
+import io.kotest.assertions.throwables.shouldThrowExactly
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.shouldBe
+
+class LottoNumberTest : FreeSpec({
+    "LottoNumber 생성" - {
+        "번호가 1 미만일 때는 예외가 발생해야한다." {
+            shouldThrowExactly<IllegalArgumentException> {
+                LottoNumber(0)
+            }.message shouldBe "로또 번호는 1에서 45 사이여야 합니다."
+        }
+        "번호가 46 이상일 때는 예외가 발생해야한다." {
+            shouldThrowExactly<IllegalArgumentException> {
+                LottoNumber(46)
+            }.message shouldBe "로또 번호는 1에서 45 사이여야 합니다."
+        }
+        "번호가 1 이상 45 이하일 때는 인스턴스가 생성되어야한다." {
+            LottoNumber(1).value shouldBe 1
+            LottoNumber(45).value shouldBe 45
```

Comment:
객체 간 비교하는건 어떨까요 ㅎㅎ
`LottoNumber`라는 친구는 값 객체로 사용할텐데, 동등함을 나타낼 수 있으면 이후 작업이 편해질 듯 해요!

```suggestion
            LottoNumber(1) shouldBe LottoNumber(1)
            LottoNumber(45) shouldBe LottoNumber(45)
```